### PR TITLE
Dw 4650 read data from htme

### PR DESCRIPTION
**Pull request by phanindrawork**

### CODE REVIEW

File: emr_jobflow_role.tf [hunk 1]
```diff
@@ -248,3 +248,54 @@ resource "aws_iam_role_policy_attachment" "analytical_dataset_generator_metadata
...
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "kms:Decrypt",
+      "kms:DescribeKey",
+    ]
+
+    resources = [
+      "${data.terraform_remote_state.ingest.outputs.s3_buckets.htme_bucket}",
```
Comment (mattburgess) [inline emr_jobflow_role.tf:287]: This should be a KMS key resource, not an S3 bucket resource.

File: emr_jobflow_role.tf [hunk 2]
```diff
@@ -248,3 +248,54 @@ resource "aws_iam_role_policy_attachment" "analytical_dataset_generator_metadata
...
+
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "s3:GetObject*",
+    ]
+
+    resources = [
+      "${data.terraform_remote_state.ingest.outputs.s3_buckets.htme_bucket}/*",
```
Comment (mattburgess) [inline emr_jobflow_role.tf:274]: Can we limit this down to just the HTME prefix/folder?

File: emr_jobflow_role.tf (hunk 1 above)
Comment (phanindrawork) [reply emr_jobflow_role.tf]: Amended Code

File: emr_jobflow_role.tf (hunk 2 above)
Comment (phanindrawork) [reply emr_jobflow_role.tf]: Amended Code

**Pull request merged**